src/Uno.UWP/Media/Capture/CameraCaptureUI.cs

```diff
@@ -0,0 +1,46 @@
+using System;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using Windows.Foundation;
+using Windows.Storage;
+
+namespace Windows.Media.Capture
+{
+	public partial class CameraCaptureUI
+	{
+		public global::Windows.Media.Capture.CameraCaptureUIPhotoCaptureSettings PhotoSettings
+		{
+			get;
+		} = new CameraCaptureUIPhotoCaptureSettings();
+
+		public global::Windows.Media.Capture.CameraCaptureUIVideoCaptureSettings VideoSettings
+		{
+			get;
+		} = new CameraCaptureUIVideoCaptureSettings();
+
+		public CameraCaptureUI()
+		{
+		}
+
+		public global::Windows.Foundation.IAsyncOperation<global::Windows.Storage.StorageFile> CaptureFileAsync(global::Windows.Media.Capture.CameraCaptureUIMode mode)
+		{
+			return AsyncOperation.FromTask(ct => CaptureFile(ct, mode));
```

Comment:
Capture of `mode` into a display class. There's no overload to prevent that?

Reply:
No other way to do this, the wrapping is required. A display class here is not a big perf hit.